cargo-apk/CHANGELOG.md

```diff
@@ -1,5 +1,7 @@
 # Unreleased
 
+- cargo-apk: Reimplement NDK r23 `-lgcc` workaround using `RUSTFLAGS`, to apply to transitive `cdylib` compilations (#TODO)
```

Comment:
the `cargo-apk:` prefix can be omitted imo

Reply:
Good catch, copy-pasted that from the PR/commit title but forgot to remove it: it should indeed be omitted as we're already in the `cargo-apk`-specific changelog :+1: